doc/classes/ProjectSettings.xml

```diff
@@ -468,12 +468,20 @@
 			Specifies the maximum number of log files allowed (used for rotation). Set to [code]1[/code] to disable log file rotation.
 			If the [code]--log-file &lt;file&gt;[/code] [url=$DOCS_URL/tutorials/editor/command_line_tutorial.html]command line argument[/url] is used, log rotation is always disabled.
 		</member>
+		<member name="debug/gdscript/warnings/accessing_underline_prefixed_member" type="int" setter="" getter="" default="1">
```

Comment:
```suggestion
		<member name="debug/gdscript/warnings/accessing_private_member" type="int" setter="" getter="" default="1">
```

I think we should name it based on what the convention is applied to, not based on the naming pattern.